src/components/SortBox/SortBox.css

```diff
@@ -1,7 +1,4 @@
-* {
-  color: white
-}
```

Comment:
Good call deleting this chunk. I had put it in temporarily just to see if I could get the text to show up and then forgot to remove it.